lib/board.rb

```diff
@@ -0,0 +1,41 @@
+require_relative './token'
+
+class Board
+    include Token
+    attr_accessor :size, :positions, :max_positions
+    
+    def initialize(size:)
+        @size = size
+        @max_positions = size * size
+        @positions = (1..max_positions).map {|num| num = empty }
+    end
+
+    def update(position:, token:)
+        positions[position] = token
+        @positions = positions;
```

Comment:
Do you want this function to update the positions attribute and return the updated positions? How do you expect to use this function? I notice that the test just tests that the function returns the updated positions array, which is fine. I don't think line 15 is necessary. If it is necessary, then you should change line 14 to be `@positions[position] = token`